vignettes/sits.Rmd

```diff
@@ -349,6 +349,110 @@ point_cf.tb <- sits_ndvi_arima_filter(point.tb, apply_whit = FALSE)
 # plot the series
 sits_merge(point_cf.tb, point.tb) %>% sits_plot()
 ```
+# Evaluating samples with self-organizing maps
+
+Conventional approaches use  only supervised classification to map LUCC, however to achieve  maps more accurated, aproaches as pre processing and pos processing must be required. sits allows to exploit a clustering method as pre-processing step using specifically self-organizing maps (SOM) to assess land cover samples of vegetation indexes time series. Independent of classifer, these techniques are suitable tools for assisting users to select representative land cover change samples from vegetation indexes time series through metrics hat indicate the sample quality and separability. 
+
+Using SOM, we can improve the performance of satellite image time series analysis for producing LUCC maps. SOM is an unsupervised neural network algorithm used to map high dimensional space onto low-dimensional space. The structure of SOM is composed by input and output layers. In the input layer contains all time series to be evaluated, also called training data. The output layer consist of a 2-D grid of neurons. Each neuron has a vector of weights representing a pattern of the input data. These neurons are organized into order in which similar pattern are closer to each other in the grid [@Kohonen2001].
+
+To train the network, the neurons are initialized randomly. The vector of weights of neuron $j$,  $w_j=[w_{j1},\ldots,w_{jn} ]$,  must have the same dimension of a input vector of time series samples $x(t)=[x(t)_1,\ldots,x(t)_n ]$. At each training step $t$, a time series sample $x(t)$ is presented to the network to find the neuron whose weight vector has the smaller distance between them. Distance metrics are utilized to compute distance $D_{j}$ between input vector $x(t)$ and each weight vector of neuron, $w_j$ for each neuron j in the output layer. The most part of literature uses Euclidean distance as metric.
```

Comment:
"between them" is confusing.

Reply:
The literature commonly uses Euclidean distance as metric because (write something here).